### DISCUSSION TITLE
Height expandable and multi-line input field

### DISCUSSION THREAD

**Discussion by thvi** [JedWatson/react-select#5026] [Q&A]
Hello,

With this component, is it possible to do this https://1drv.ms/u/s!Aj16T_Ds0bkv2zkeGHuKes9N-cGM ?

Bests

**Comment by Rall3n** [comment]
Hello @thvi,

you can do this with the `formatOptionLabel` prop. It allows you to format the label in menu options and the control as React components.

```jsx
// Assuming options are of shape {value, label, street, city, zip}

<Select 
  { ... }
  formatOptionLabel={(option, meta) => <Fragment>
    <div>{option.label}</div>
    <div>{option.street}<br />{option.zip}, {option.city}</div>
  </Fragment}
/>
```
